charts/logging-operator/templates/logging.yaml

```diff
@@ -20,7 +20,8 @@ spec:
   {{- end }}
   {{- if (not .Values.logging.fluentdDisabled) }}
   {{- if .Values.logging.fluentd }}
-  fluentd: {{- .Values.logging.fluentd . | nindent 4 }}
+  fluentd: 
+    {{- toYaml .Values.logging.fluentd | nindent 4 }}
```

Comment:
why would you put it on a new line? nindent should do that for us

Reply:
just a nit, lgtm anyways

Reply:
I copy pasted a line below. Just updated the whole file :)